content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -170,6 +257,53 @@ metadata:
 type: Opaque
 ```
 
+Generated Secrets can be used in Deployments by refering to the name of the secretGenerator:
+
+```yaml
```

Comment:
Either you use `yaml` and `none` separately for the following two files, or you can wrap the content using `shell`. This content currently is not valid YAML.

Reply:
You're right, good catch. Fixed.